networking/aws_load_balancer_operator/configure-egress-proxy-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,11 @@
+:_content-type: ASSEMBLY
+[id="nw-aws-load-balancer-operator-cluster-wide-proxy"]
+= Configuring cluster-wide proxy
+:context: aws-load-balancer-operator
+
+toc::[]
+
+You can configure the cluster-wide proxy in the AWS Load Balancer Operator.
```

Comment:
is it possible that you can add more details or elaborate? This statement looks redundant to me.

Reply:
Yes. we can elaborate. Addressed